### PR TITLE
[Doc] Add Superedge and Wasmedge integration document

### PR DESCRIPTION
**Pull request by malc0lm**
Fix: https://github.com/WasmEdge/WasmEdge/issues/598

### CODE REVIEW

File: docs/book/en/src/kubernetes/kubernetes/superedge.md [hunk 1]
```diff
@@ -1,3 +1,148 @@
...
+
+Use the simple install script to install WasmEdge on your edge node.
+
+```bash
+curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash
+```
+
+## Build And install Crun with WasmEdge
+
+The [crun](https://github.com/containers/crun) project has WasmEdge support baked in. For now, the easiest approach is just built it yourself from source. First, let's make sure that crun dependencies are installed on your Ubuntu 20.04. For other Linux distributions, please see [here](https://github.com/containers/crun#readme).
```
Comment (hydai) [inline docs/book/en/src/kubernetes/kubernetes/superedge.md:41]: `built` -> `to build`.

File: docs/book/en/src/kubernetes/kubernetes/superedge.md [hunk 2]
```diff
@@ -1,3 +1,148 @@
...
+    imagePullPolicy: IfNotPresent
+    name: wasi-demo
+  hostNetwork: true
+  restartPolicy: Never
+EOF
+
+kubectl create -f wasmedge-app.yaml
+```
+
+The output will showed by executing `kubectl logs wasi-demo` command.
```
Comment (hydai) [inline docs/book/en/src/kubernetes/kubernetes/superedge.md:136]: `showed` -> `show`

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/book/en/src/kubernetes/kubernetes/superedge.md (hunk 2 above)
Comment (malc0lm) [reply docs/book/en/src/kubernetes/kubernetes/superedge.md]: tks, done.

File: docs/book/en/src/kubernetes/kubernetes/superedge.md (hunk 1 above)
Comment (malc0lm) [reply docs/book/en/src/kubernetes/kubernetes/superedge.md]: tks, done.

**Review by hydai — APPROVED** [review]

**Pull request merged**